fdbserver/VersionedBTree.actor.cpp

```diff
@@ -871,17 +886,21 @@ class FIFOQueue {
 	}
 
 	// Fast path extent peekAll (this zooms through the queue reading extents at a time)
-	ACTOR static Future<Standalone<VectorRef<T>>> peekAll_ext(FIFOQueue* self) {
+	//ACTOR static Future<Standalone<VectorRef<T>>> peekAll_ext(FIFOQueue* self,
```

Comment:
Can you add a comment describing what is sent to the output stream - is it one vector of results per extent found?  And the terminating condition?

Reply:
Done. It's a vector of results per extent.